api/server/methods.ts

```diff
@@ -75,14 +76,13 @@ export function initMethods() {
       if (!chatExists) throw new Meteor.Error('chat-not-exists',
         'Chat doesn\'t exist');
 
-      return {
-        messageId: Messages.collection.insert({
-          senderId: this.userId,
-          chatId: chatId,
-          content: content,
-          createdAt: new Date()
-        })
-      }
+      Messages.collection.insert({
+        senderId: this.userId,
+        chatId: chatId,
+        content: content,
```

Comment:
I first upload the picture using the `pictureService` and once finished I send a message. Wish it could be co-related but it has to be done separately because the server can't access client FS unless he uploaded his content.